.goreleaser.yml

```
@@ -1,47 +1,75 @@
 archives:
   - files:
-      - none*
+      # Ensure only built binary is archived
+      - 'none*'
     format: zip
     name_template: '{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}'
 before:
   hooks:
-    - go mod download
+    - 'go mod download'
 builds:
-  - binary: '{{ .ProjectName }}_{{ .Version }}'
+  - # Binary naming only required for Terraform CLI 0.12
+    binary: '{{ .ProjectName }}_v{{ .Version }}_x5'
+    env:
+      - CGO_ENABLED=0
     flags:
       - -trimpath
-    goarch:
-      - '386'
-      - amd64
-      - arm
-      - arm64
     goos:
       - darwin
       - freebsd
       - linux
       - windows
+    goarch:
+      - '386'
+      - amd64
+      - arm
+      - arm64
     ignore:
-      - goarch: '386'
-        goos: darwin
+      - goarch: arm
+        goos: windows
+      - goarch: arm64
+        goos: freebsd
+      - goarch: arm64
+        goos: windows
     ldflags:
-      - -s -w -X version.ProviderVersion={{.Version}}
+      - -s -w -X main.Version={{.Version}}
```

Reverting this back to `version.ProviderVersion` 
```suggestion
      - -s -w -X version.ProviderVersion={{.Version}}
```